src/test/rpc/DepositAuthorized_test.cpp

```diff
@@ -276,11 +285,246 @@ class DepositAuthorized_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testCredentials()
+    {
+        using namespace jtx;
+
+        const char credType[] = "abcde";
+
+        Account const alice{"alice"};
+        Account const becky{"becky"};
+        Account const carol{"carol"};
+
+        Env env(*this);
+        env.fund(XRP(1000), alice, becky, carol);
+        env.close();
+
+        // carol recognize becky
```

Comment:
This comment seems incorrect, the credential has nothing to do with Becky.

Reply:
fixed